### ISSUE TITLE
Deprecated attribute "title"

### ISSUE THREAD

**Issue by Tejas2805** [se-edu/se-book#82]
The attribute "title" used for popovers and modals is deprecated. It may not be used in the future. Better to replace "title" with header"

**Comment by damithc** [comment]
True. I'm waiting till the semester is over to fix this. As the book is used in other modules. I don't want to cause unnecessary updates (and merge conflicts) to the shared code.

**Comment by Tejas2805** [comment]
Sure! Lmk when can. Will help out.
